packages/Popover/src/components/Content/Content.js

```diff
@@ -41,59 +47,63 @@ class Content extends React.Component {
     setTimeout(() => {
       if (!isElementContainsFocus(currentTarget)) {
         onClose();
+        onBlur();
       }
     }, parseInt(PopoverConstants.transition, 10));
   };
 
-  // TODO: extract this to Storybook story somehow so supporting numbers as strings is not required
-  isNumber = n => RegExp(/^[0-9]+$/).test(n);
+  const {
+    content,
+    isEager,
+    isOpen,
+    onClose,
+    onDelayedClose,
+    onDelayedOpen,
+    portalElement,
+    refContent,
+  } = React.useContext(PopoverContext);
 
-  render() {
-    return (
-      <PopoverContext.Consumer>
-        {({ content, isEager, isOpen, onClose, onDelayedClose, onDelayedOpen, portalElement, refContent }) => {
-          const contentStyles = {
-            left: content.x,
-            maxWidth: this.isNumber(content.maxWidth) ? `${content.maxWidth}px` : content.maxWidth,
-            top: content.y,
-            width: content.width,
-          };
+  const handleRef = _ref => {
+    refContent(_ref);
```

Comment:
I have to change as well the context because is receiving a function, not feel 💯 confident that the popover has enough testing currently for doing the refactor in this branch.